app/controllers/pieces_controller.rb

```diff
@@ -48,7 +48,9 @@ def message
   end
 
   def attempt_move
-    selected_piece.move_to!(destination_coordinates) if moving_validly?
+    return unless moving_validly?
+    selected_piece.move_to!(destination_coordinates)
+    Pusher.trigger(current_game.id, 'refresh_event', message: '') unless Rails.env.test?
```

Comment:
:+1: Looks good.

I forsee a potential situation where multiple developers are testing a game with the same id in the dev environement and refreshing each other's pages :smile: